### PR TITLE
Fix #120 - Accept callback on subsReady.

### LINKED ISSUE #120: Accept callback for `subsReady`
This is very useful for events where we don't have to run a autorun. 
See this issue: https://github.com/meteorhacks/flow-router/issues/119#issuecomment-104836746

Here's the suggest API:

~~~js
Template.myTemplate.events(
   'click #id': function(){
      FlowRouter.subsReady("myPost", function() {
         // do something
      });
  }
)
~~~

### PR DESCRIPTION
**Pull request by delgermurun**
Add support for #120

### CODE REVIEW

**Comment by delgermurun** [comment]
@arunoda Please check this out.

File: client/router.js [hunk 1]
```diff
@@ -198,6 +198,12 @@ Router.prototype.ready = function() {
 };
 
 Router.prototype.subsReady = function() {
+  var callback, args = Array.prototype.slice.call(arguments);
```
Comment (arunoda) [inline client/router.js]: This seems like `Array.slice` returning two items :) Better to write like this.

~~~js
var callback = null;
var args = Array.prototype.slice.call(arguments);
~~~

And how about using `_.toArray()` instead `Array.slice`

File: client/router.js [hunk 2]
```diff
@@ -198,6 +198,12 @@ Router.prototype.ready = function() {
...
+
+  if (typeof args[args.length - 1] === "function") {
```
Comment (arunoda) [inline client/router.js]: How about using `_.last()`
Then it seems give very readable code.

**Comment by delgermurun** [comment]
@arunoda updated.

**Comment by arunoda** [comment]
Nice. I'll take this and update docs :)

**Comment by arunoda** [comment]
Released with `v1.9.0`

**Pull request merged**